packages/web/src/App.tsx

```diff
@@ -47,7 +47,7 @@ class App extends Component {
                 <Route exact path="/about" component={About} />
                 <Route path="/coordinated-entry" component={CoordinatedEntry} />
                 <Route path="/shelters" component={Shelters} />
-                <Route path="/job-training" component={JobTraining} />
+                <Route path="/job_training" component={JobTraining} />
```

Comment:
Ideally we'll update this to pull dynamically from the DB instead but I wanted to get this in quickly for release.